core/modules/field_group/field_group.api.php

```diff
@@ -0,0 +1,372 @@
+ *
+ * DEVELOPERS NOTES
+ *
+ * - Field group uses a '#fieldgroups' property to know what field groups are to
+ *   be pre_rendered and rendered by the field_group module. '#fieldgroups' is
+ *   later merged with the normal #groups that can be used by any other module.
+ */
+
+/**
+ * @addtogroup hooks
+ * @{
+ */
+
+
+/**
+ * Javascript hooks
+ *
+ * Backdrop.FieldGroup.Effects.processHook.execute()
+ * See field_group.js for the examples for all implemented formatters.
+ */
+
+
+/**
+ * Define the information on formatters. 
+ *
+ * The formatters are separated by dispalay mode type. Use "form" for all form 
+ * elements and "display" will be the real display modes (full, teaser, etc).
+ *
+ * structure:
+ * @code
+ * array(
+ *   'form' => array(
+ *     'fieldset' => array(
+ *       // required, String with the name of the formatter type.
+ *       'label' => t('Fieldset'),
+ *       // optional, String description of the formatter type.
+ *       'description' => t('This is field group that ...'),
+ *       // required, Array of available formatter options.
+ *       'format_types' => array('open', 'collapsible', 'collapsed'),
+ *       // required, String with default value of the style.
+ *       'default_formatter' => 'collapsible',
+ *       // optional, Array with key => default_value pairs.
+ *       'instance_settings' => array('key' => 'value'),
+ *     ),
+ *   ),
+ *   'display' => array(
+ *     'fieldset' => array(
+ *       // required, String with the name of the formatter type.
+ *       'label' => t('Fieldset'),
+ *       // optional, String description of the formatter type.
+ *       'description' => t('This is field group that ...'),
+ *       // required, Array of available formatter options.
+ *       'format_types' => array('open', 'collapsible', 'collapsed'),
+ *       // required, String with default value of the style.
+ *       'default_formatter' => 'collapsible',
+ *       // optional, Array with key => default_value pairs.
+ *       'instance_settings' => array('key' => 'value'),
+ *     ),
+ *   ),
+ * ),
+ * @endcode
+ *
+ * @return $formatters
+ *   A collection of available formatting html controls for form
+ *   and display overview type.
+ *
+ * @see field_group_field_group_formatter_info()
+ */
+function hook_field_group_formatter_info() {
+  return array(
+    'form' => array(
+      'fieldset' => array(
+        'label' => t('Fieldset'),
+        'description' => t('This field group renders the inner content in a fieldset with the title as legend.'),
+        'format_types' => array('open', 'collapsible', 'collapsed'),
+        'instance_settings' => array('classes' => ''),
+        'default_formatter' => 'collapsible',
+      ),
+    ),
+    'display' => array(
+      'div' => array(
+        'label' => t('Div'),
+        'description' => t('This field group renders the inner content in a simple div with the title as legend.'),
+        'format_types' => array('open', 'collapsible', 'collapsed'),
```

Comment:
Do these even make sense for a div?

Reply:
no? I'll check and see if they are used anywhere (or do anything!)

Reply:
This was also in the API file, so I swapped the examples out with fieldset for both form and display. 

I also checked the actual definition of a div in the module file, and it also has these 'format_types` defined, but they are not used and do not do anything, so I have removed them. I suspect it was a copy/paste error when `div` was added.

Reply:
Yes, there are also other strange settings for `div` like "blind", "effect" and "speed". I have no idea what any of these are, but we should add description text when we figure it out :)